build-system/pr-check/dist-tests.js

```diff
@@ -44,6 +45,7 @@ function main() {
 
   if (!isPullRequestBuild()) {
     downloadDistOutput(FILENAME);
+    downloadEsmDistOutput(FILENAME);
```

Comment:
I see that you've added `test-double-loading.js`. Does this change mean that both esm and non-esm minified binaries must be available before one can run the non-esm integration tests? Also, how will this affect integration tests on unminified binaries where there's only one flavor?

At first glance, I think this is okay for CI runs. However, we shouldn't enforce the building of both esm and non-esm for development. Perhaps there's a way to get `test-double-loading.js` to be a no-op when only one binary is available?

Reply:
agree. i'll try and make it so that it only executes on the module tests